lab_2_keywords_cooccurrence/main.py

```diff
@@ -40,7 +138,13 @@ def calculate_frequencies_for_content_words(candidate_keyword_phrases: KeyPhrase
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if not check_keyphrases(candidate_keyword_phrases):
+        return None
+    freq_dict = {}
+    for phrases in candidate_keyword_phrases:
+        for words in phrases:
```

Comment:
can you optimize using `phrases.count` ?

Reply:
Done